README.md

```diff
@@ -75,11 +75,14 @@ Consult the [LORIS Wiki](https://github.com/aces/Loris/wiki/Setup) page on this
 5. Apache configuration
 
 If your apache configuration was not completed by the Install script, run the following enable rewriting of LORIS, enable your `$projectname` site, and restart apache:  (run by user who has root privileges)
+    
 ```bash
     sudo a2enmod rewrite
     sudo a2dissite default
     sudo a2ensite $projectname
     sudo service apache2 reload
+    ```
```

Comment:
I think this has to be at the same indentation level as the "bash" line

Reply:
ok.  just committed again.  re-check?